components/gfx/font_cache_thread.rs

```diff
@@ -216,7 +216,8 @@ impl FontCache {
                     url: url.clone(),
                     type_: RequestType::Font,
                     destination: Destination::Font,
-                    origin: url.clone(),
+                    // TODO: Add a proper origin - Can't import GlobalScope from gfx
```

Comment:
Instead of using `GlobalScope`, figure out a way to directly thread through an `Origin`?